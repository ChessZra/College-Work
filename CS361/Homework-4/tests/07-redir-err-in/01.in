ls 361 2>file.err
cat file.err
rm -v file.err
