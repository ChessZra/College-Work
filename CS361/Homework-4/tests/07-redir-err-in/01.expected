cs361sh> cs361sh> ls: cannot access '361': No such file or directory
cs361sh> removed 'file.err'
cs361sh> 
